src/valid/error.rs

```diff
@@ -9,20 +9,20 @@ pub struct ValidationError<E>(Vec<Cause<E>>);
 
 impl<E: Display> Display for ValidationError<E> {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        f.write_str("Validation Error\n")?;
+        f.write_str("Invalid Configuration\nCaused by:\n")?;
         let errors = self.as_vec();
         for error in errors {
-            f.write_str(format!("{} {}", '\u{2022}', error.message).as_str())?;
+            f.write_str(format!("   {} {}", '\u{2022}', error.message).as_str())?;
             if !error.trace.is_empty() {
                 f.write_str(
                     &(format!(
-                        " [{}]",
+                        " [at schema.{}]",
                         error
                             .trace
                             .iter()
                             .cloned()
                             .collect::<Vec<String>>()
-                            .join(", ")
+                            .join(".")
```

Comment:
I think you got the problem! We are wrapping ValidationError in a anyhow::Error somewhere, which forcefully converts the ValidationError into a String using this Display implementation, and then wraps it inside of anyhow::Error.

The solution should make sure that we don't convert a ValidationError into a String. Infact we should drop the Display implementation of ValidationError.

Reply:
Let me check